channeldb/revocation_log.go

```diff
@@ -238,13 +258,18 @@ type RevocationLog struct {
 	// all revocation logs.
 	TheirBalance tlv.OptionalRecordT[tlv.TlvType4, tlv.BigSizeT[lnwire.MilliSatoshi]] //nolint:lll
 
+	// CustomBlob is an optional blob that can be used to store information
+	// specific to a custom channel type. This information is only created
```

Comment:
Is this comment accurate for a revocation log entry? Isn't this information created for each commitment transaction? So not at channel funding time only?